CHANGES.rst

```diff
@@ -1,13 +1,47 @@
 Changelog
 =========
 
-Version 0.8.0 [unreleased]
+Version 1.0.0 [2022-02-18]
 --------------------------
 
+Features
+~~~~~~~~
+
+- Added `customizable navigation menu <https://github.com/openwisp/openwisp-utils#main-navigation-menu>`_
+- Added `horizontal filters <https://github.com/openwisp/openwisp-utils#admin-filters>`_
+- Added `customizable admin dashboard <https://github.com/openwisp/openwisp-utils#openwisp-dashboard>`_
+- Added `send_email function <https://github.com/openwisp/openwisp-utils#openwisp_utilsadmin_themeemailsend_email>`_
+- Added `CompressStaticFilesStorage <https://github.com/openwisp/openwisp-utils#openwisp_utilsstoragecompressstaticfilesstorage>`_ -
+  a static storage backend for Django that also compresses static files.
+- Added `AssertNumQueriesSubTestMixin <https://github.com/openwisp/openwisp-utils#openwisp_utilstestsassertnumqueriessubtestmixin>`_
+- Added `HelpTextStackedInline admin class <https://github.com/openwisp/openwisp-utils#openwisp_utilsadminhelptextstackedinline>`_
+- Added `OpenwispCeleryTask <https://github.com/openwisp/openwisp-utils#openwisp-utils-tasks-openwispcelerytask>`_ - a custom celery task class
+- Added support for linting CSS and JS in `openwisp-qa-check <https://github.com/openwisp/openwisp-utils#openwisp-qa-check>`_
+- Added support for formatting CSS and JS in `openwisp-qa-format <https://github.com/openwisp/openwisp-utils#openwisp-qa-format>`_
+- Added `git pre-push hook <https://github.com/openwisp/openwisp-utils/issues/161>`_
+
 Changes
 ~~~~~~~
 
-- Updated Django REST Framework to 3.12.x
+- `Updated OpenWISP's admin theme <https://medium.com/@niteshsinha1707/new-navigation-menu-and-ui-ux-improvements-project-report-a94c37514b7d>`_
+- Added support for Django 3.2.x and 4.0.x
+- Added support for Python 3.9
```

Comment:
I am not sure if these should go under features or dependencies

Reply:
in dependencies